<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Shaunenoch" failures="5" tests="27" name="runner.TestRunner" time="84.593" errors="0" timestamp="2023-05-20T10:09:13 EDT" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="3.491"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.524"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.259"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.738">
    <failure message="expected [Welcome to Queue] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Welcome to Queue] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.QueueSteps.user_should_get_a_error_message(QueueSteps.java:134)
at ✽.user should get a error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject3/src/test/resources/FeaturesDsAlgo/Queue.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.407">
    <failure message="expected [Welcome to Graph] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Welcome to Graph] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.GraphSteps.user_should_have_error_message(GraphSteps.java:133)
at ✽.user should have error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject3/src/test/resources/FeaturesDsAlgo/Graph.feature:25)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.417"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.299"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.491"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="7.575"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.099">
    <failure message="expected [U are logged in] but found [Invalid Username and Password]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [U are logged in] but found [Invalid Username and Password]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.LoginSteps.user_stays_on_the_login_page(LoginSteps.java:182)
at ✽.user stays on the login page(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject3/src/test/resources/FeaturesDsAlgo/ALogin.feature:47)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="5.072"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="5.219">
    <failure message="expected [password_mismatch:The two password fields didn’t match.] but found [U are logged in]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password_mismatch:The two password fields didn’t match.] but found [U are logged in]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.LoginSteps.user_prints_the_error_message(LoginSteps.java:55)
at ✽.user prints the error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject3/src/test/resources/FeaturesDsAlgo/ALogin.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.510"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.373"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.026">
    <failure message="expected [] but found [Overview of Trees]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [Overview of Trees]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at Steps_DsAlgo.TreeSteps.user_checks_for_the_error_message(TreeSteps.java:84)
at ✽.user checks for the error message(file:///C:/Users/shaun/eclipse-workspace/Ds_AlgoProject3/src/test/resources/FeaturesDsAlgo/FTree.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.929"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.244"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="6.217"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.034"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.284"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.484"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.403"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.860"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.536"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.529"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.490"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.083"/>
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
